fast64_internal/mk64/mk64_actors.py

```diff
@@ -0,0 +1,200 @@
+import math, bpy, mathutils
+import os
+from bpy.utils import register_class, unregister_class
+from ..panels import MK64_Panel
+
+
+
+enumModelIDs = [
+    ("Custom", "Custom", "Custom"),
+    ("ITEMBOX", "itembox", "itembox"),
+    ("FOLIAGE", "foliage", "foliage"),
+    ("FALLING_ROCK", "fallingRocks", "fallingRocks"),
+    ("PALM_TREES", "palmTrees", "palmTrees"),
+    ("PIRANHA_PLANT", "piranhaPlant", "piranhaPlant"),
+    ("Cow", "cow", "cow"),
+]
+
+class SearchModelIDEnumOperator(bpy.types.Operator):
+    bl_idname = "object.search_model_id_enum_operator"
```

Comment:
not sure how it is on sm64-side but make sure you don't use the same ID twice here (for instance on OoT we're using ``object.oot_thing_operator``)

note: I don't know if this is actually an issue but I guess it's better to be careful